packages/indicators/src/__tests__/builders/buildArithmetic/buildArithmetic.test.ts

```diff
@@ -0,0 +1,149 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { ArithmeticConfig, buildArithmetic } from '../../../builders/buildArithmetic';
+import { AnalyticValue } from '../../../types';
+import { createAggregator } from '../stubs';
+import { ANALYTIC_RESPONSE_FIXTURES } from './buildArithmetic.fixtures';
+
+const aggregationsByCode = (codes: string[]) =>
+  Object.fromEntries(codes.map(code => [code, ['FINAL_EACH_YEAR']]));
```

Comment:
This functions' name is a bit unorthodox: typically it should be `getAggregationsByCode`, but I like the declarativeness of simply using a noun. I feel that the parentheses and syntax highlighting make this unambiguous and safe to use, at least in the context of unit test data creation.

You may disagree with that, in that case I'll be happy to change this to `getAggregationByCode`. But I thought I should give it a try first 😄

Reply:
I actually do disagree! Even knowing about it from this comment, I still found it took me an extra moment when I was reading through the first test to get it. But this is not an autocracy, so if you can get someone else in your camp, you're welcome to use this name. 

My pick would be even more specific: `getFinalEachYearConfigForCodes`

Reply:
Well using a `get...` prefix is standardised in software nomeclature. I've seen instances of not using it under specific conditions and I like it, but it's something I would introduce only if there is a consensus.